src/android/io/sentry/SentryCordova.java

```diff
@@ -207,12 +211,36 @@ private void startWithOptions(final JSONObject jsonOptions, final CallbackContex
     }
   }
 
-  private void captureEnvelope(String envelope, final CallbackContext callbackContext) {
-    if (envelope == null || envelope.equals("")) {
+  private void captureEnvelope(String headerString, String payloadString, String payloadType, final CallbackContext callbackContext) {
```

Comment:
Can you get around from passing the header?
This method can create a header too, you could provide the `sdk.name` to the bridge when initializing things?

Reply:
requires event id and sdk object which contains more than just a name.
at this point, I think it's good the way as it is otherwise we'll be adding one more level of serialization/deserialization.